openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash -eux
+#
+# Run Charmed Openstack CI tests manually in a similar way to how they are run
+# by OpenStack CI (OSCI).
+#
+# Usage: clone/fetch charm to test and run from within charm root dir.
+#
+FUNC_TEST_PR=
+FUNC_TEST_TARGET=
+SKIP_BUILD=false
+
+usage () { echo "USAGE: --func-test-target | --func-test-pr | --skip-build"; }
```

Comment:
Ideally this would make it more clear how to use the tool, e.g. that you have to call it from within a charm directory, etc.

Reply:
i have put in a much nicer help message